### PR TITLE
fix: Downgrade RootParticleWriter log message

### PR DESCRIPTION
**Pull request by paulgessinger**
In ttbar / g4 jobs especially, we have **many** particles without hits. The `INFO` level seems excessive for these cases.

### CODE REVIEW

**Review by andiwand — APPROVED** [review]

**Comment by andiwand** [comment]
alternatively we could also fill 0 in this case I guess

File: Examples/Io/Root/src/RootParticleWriter.cpp [hunk 1]
```diff
@@ -186,8 +186,8 @@ ActsExamples::ProcessCode ActsExamples::RootParticleWriter::writeT(
       // get the sim hits
       auto it = hitsPerParticle.find(particle.particleId());
       if (it == hitsPerParticle.end()) {
-        ACTS_INFO("Could not find sim hits for "
-                  << particle.particleId() << " in event " << ctx.eventNumber);
+        ACTS_DEBUG("Could not find sim hits for "
+                   << particle.particleId() << " in event " << ctx.eventNumber);
```
Comment (andiwand) [inline Examples/Io/Root/src/RootParticleWriter.cpp:190]:
```suggestion
        ACTS_DEBUG("Could not find sim hits for "
                   << particle.particleId() << " in event " << ctx.eventNumber);
        m_numberOfHits.push_back(0);
```

**Comment by paulgessinger** [comment]
Output changed due to writing a 0. Can you go again @andiwand ?

**Review by andiwand — APPROVED** [review]
thanks!

**Comment by acts-project-service** [comment]
## :white_check_mark: Athena integration test results [fc754692a02f21804fd215e67886631feea3e509]
### :white_check_mark: All tests successful

|status|job|report|
|------|---|------|
|:green_circle: | [run_unit_tests](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307986)||
|:green_circle: | [test_ActsEFTrackFit](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307985)||
|:green_circle: | [test_ActsBenchmarkWithSpot](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307984)||
|:green_circle: | [test_ActsConversionWorkflow](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307983)||
|:green_circle: | [test_ActsWorkflow](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307982)||
|:green_circle: | [test_ActsValidateAmbiguityResolution](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307981)||
|:green_circle: | [test_ActsValidateResolvedTracks](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307980)||
|:green_circle: | [test_ActsValidateTracks](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307979)||
|:green_circle: | [test_ActsValidateActsCoreSpacePoints](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307978)||
|:green_circle: | [test_ActsValidateActsSpacePoints](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307977)||
|:green_circle: | [test_ActsValidateSeeds](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307976)||
|:green_circle: | [test_ActsValidateOrthogonalSeeds](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307975)||
|:green_circle: | [test_ActsValidateClusters](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307974)||
|:green_circle: | [test_ActsPersistifyEDM](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307973)||
|:green_circle: | [test_ActsGSFRefitting](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307972)||
|:green_circle: | [test_ActsKfRefitting](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307971)||
|:green_circle: | [test_ActsExtrapolationAlgTest](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307970)||
|:green_circle: | [test_ActsITkTest](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307969)||
|:green_circle: | [run_workflow_tests_run4_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307968)||
|:green_circle: | [run_workflow_tests_run2_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307967)||
|:green_circle: | [run_workflow_tests_run2_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307966)||
|:green_circle: | [run_workflow_tests_run3_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307965)||
|:green_circle: | [run_workflow_tests_run3_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307964)||
|:green_circle: | [run_art_test: test_data18_13TeV_1000evt](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307963)||
|:green_circle: | [run_art_test: test_ttbarPU40_reco](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36307962)||

**Pull request merged**